app/screens/Info.js

```diff
@@ -1,17 +1,49 @@
 import React from 'react'
-import { SafeAreaView, StyleSheet, ScrollView, View } from 'react-native'
+import {
+  SafeAreaView,
+  StyleSheet,
+  ScrollView,
+  View,
+  FlatList,
+} from 'react-native'
 import { colors } from 'app/config'
+import ListHeader from '../components/common/ListHeader'
 import ListItem from '../components/common/ListItem'
+import { useSelector } from 'react-redux'
+import { selectRackingsByIds } from '../store/reducers/rackings'
+import { selectSystemById } from '../store/reducers/systems'
 
-const Info = ({ navigation }) => {
+const Info = ({ navigation, route }) => {
+  const { systemId, rackingChildren } = route.params
+  const rackings = useSelector(selectRackingsByIds(rackingChildren))
+  const system = useSelector(selectSystemById(systemId))
+  const renderItem = ({ item }) => {
+    return (
+      <ListItem
+        title={item.fields.name}
+        subTitle={item.fields.short_description}
+        onPress={() =>
+          navigation.navigate('Detail', {
+            id: item.id,
+            title: item.fields.name,
+          })
+        }
+      />
+    )
+  }
   return (
     <SafeAreaView style={styles.container}>
       <ScrollView
         contentInsetAdjustmentBehavior="automatic"
         style={styles.scrollView}
       >
         <View style={styles.container}>
-          <ListItem onPress={() => navigation.navigate('Detail')} />
+          <ListHeader images={system.fields.header_image} />
```

Comment:
Again you don't need the SrollView if the FlatList is handling listing out the content for the screen.

Have a look at this https://reactnative.dev/docs/flatlist#listheadercomponent
That will allow you to add a header component to the FlatList and will handle scrolling the header and list content.